common/static/js/spec_helpers/rwd_header_footer.js

```diff
@@ -1,99 +0,0 @@
-/**
```

Comment:
Note to reviewers: renamed this `rwd_header.js` because the new footer is always responsive.